bzl/ppx/exe/BUILD.bazel

```diff
@@ -0,0 +1,1336 @@
+load("@obazl_rules_ocaml//ocaml:rules.bzl", "ppx_executable", "ppx_module")
```

Comment:
I am still really not a fan of enumerating all of our possible combinations of ppx libraries as targets.

Is there a way to autogenerate this file, or better still to elide it completely?